src/api/app/views/webui2/shared/_requests_state_and_type_form.html.haml

```diff
@@ -0,0 +1,13 @@
+%p
+  Display
+  = select_tag('request_type_select',
```

Comment:
You forgot the `.form-control` class. Also the whole thing is rather crowded and unaligned. 

![screen shot 2018-08-22 at 00 45 59](https://user-images.githubusercontent.com/514785/44433015-d31e5480-a5a4-11e8-8d62-6435fcf6d8d8.png)